app/assets/v2/js/pages/profile.js

```diff
@@ -10,8 +10,30 @@ $(document).ready(function() {
 
   setupTabs('#activity-tabs');
 
+  let file = '';
+
+  $('body').on('click', 'img.picky', function() {
+    $('.selected').removeClass('selected');
+    $(this).addClass('selected');
+    let link = $(this).attr('src');
+
+    srcToFile(link, $(this).attr('alt'), 'image/jpeg').then(result => {
```

Comment:
we can remove the file upload functionality from this PR, right?  since the headers are all already uploaded..

Reply:
This converts the uploaded images to file object for upload.